src/ui.form-input-file/index.vue

```diff
@@ -88,6 +88,12 @@ import { useLocale } from "../composable.locale";
 import { UInputFile } from "./constants";
 import defaultConfig from "./configs/default.config";
 
+import UText from "../ui.text-block";
+import UIcon from "../ui.image-icon";
+import ULabel from "../ui.form-label";
+import UButton from "../ui.button";
+import UFiles from "../ui.text-files";
```

Comment:
In all components this imports goes with other non-local imports (before or after UIService)